src/main/java/nl/uu/cs/ape/domain/BioToolsAPI.java

```diff
@@ -71,30 +86,34 @@ public static JSONArray readListOfTools(String filePath) throws IOException {
 	 * @return JSONArray with the tool annotations.
 	 * @throws IOException - If the file cannot be read or written.
 	 */
-	public static JSONArray getToolsFromDomain(String domainName) throws IOException {
+	private static JSONObject getToolsFromDomain(String domainName) throws IOException {
 		JSONArray toolAnnotations = null;
 		if (!domainName.equals("")) {
 			toolAnnotations = fetchToolsFromURI("https://bio.tools/api/t?domain=" + domainName + "&format=json");
 		} else {
 			toolAnnotations = fetchToolsFromURI("https://bio.tools/api/t?format=json");
 		}
-		return toolAnnotations;
+		return convertBioTools2Ape(toolAnnotations, true);
 	}
 
 	/**
-	 * Send Get request to get tool annotations Saves JSONArray with all
-	 * bio.tools that belong to a certain EDAM topic.
+	 * Retrieve tools from the bio.tools API for a given topic and convert them to
+	 * the tool annotation format used by the APE library.
 	 * 
-	 * @throws IOException
+	 * @param topicName The name of the topic.
+	 * @return The JSONObject containing the tool annotations in the APE format.
+	 * 
```

Comment:
Is the blank line between "@return" and "@throws" by purpose?

Reply:
I removed it, thanks!